MAX_ACCELERATION: 6000
MAX_DECELERATION: 6000
PROFILE_VELOCITY: 6000